python/lancedb/db.py

```diff
@@ -104,6 +163,7 @@ def __contains__(self, name: str) -> bool:
     def __getitem__(self, name: str) -> LanceTable:
         return self.open_table(name)
 
+    @_not_supported_on_remote
```

Comment:
How can user to create table when they are using remote server ? :)

Reply:
right now create table is not supported over the remote. we can add that api.